src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Square implements Figure {
+    private int side;
+    private String color;
+    public Square() {
```

Comment:
The default constructor should not be empty or assigning values as it does now. If the intention is to create a `Square` with specific `side` and `color`, you should define a constructor that accepts these parameters.